.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: shell
+notifications:
+  email:
+    on_success: never
+stages:
+- tests
+- deployment
+env:
+  global:
+    - RELEASE_REPO_TO_FIND="https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/releases/download/latest/"
+    - RELEASE_REPO_TO_REPLACE="https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/releases/download/$TRAVIS_TAG/"
+jobs:
+  include:
+    - stage: tests
+      name: Tests & Linting
+      language: shell
+      services: docker
+      script:
+        - docker build -f pipeline/Dockerfile -t lambda_image .
+        - docker run --rm lambda_image bash -c 'python3 -m pylint --rcfile=pipeline/pylint.cfg $(find src/ -name "*.py")'
+        - docker run --rm lambda_image bash -c 'cd src && python3 -m pytest -v ../tests/unit'
+    - stage: deployment
+      name: Github Release Deployment
+      if: tag =~ /^release.*$/

```

Comment:
summary from what you explained: new tag "release.*" triggers entire travis build + as this condition is true, also release package is created and lands in here:
https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/releases

Afterwards one-liner should be adapted to use latest release